src/components/Patient/DailyRounds.tsx

```diff
@@ -788,6 +788,30 @@ export const DailyRounds = (props: any) => {
                   },
                 ]}
               />
+              <RangeAutocompleteFormField
```

Comment:
There's already an SpO2 field, [line 848](https://github.com/ohcnetwork/care_fe/blob/5d2d2849b6267b7e38cc57c2f4083c5b38390c2e/src/components/Patient/DailyRounds.tsx#L848) onwards that's used for other rounds type, let's move that out to the earlier block, after line [762.](https://github.com/ohcnetwork/care_fe/blob/5d2d2849b6267b7e38cc57c2f4083c5b38390c2e/src/components/Patient/DailyRounds.tsx#L762)

Reply:
I did the same

Reply:
Yea, but now we have SpO2 field specified twice, when we can achieve the same result with one.

Reply:
So we dont need SpO2 field for community nurses log update ?

Reply:
We do.

Please check the file itself, the code block that starts at [726](https://github.com/ohcnetwork/care_fe/blob/5d2d2849b6267b7e38cc57c2f4083c5b38390c2e/src/components/Patient/DailyRounds.tsx#L726) includes fields that should appear for all 4 round types (that should have SpO2 field under new requirements).

So you can move the code starting at line 848 there, and just remove the code you added.